docs/control-property-schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://raw.githubusercontent.com/microsoft/PowerApps-Tooling/master/docs/control-type-schema.json",
```

Comment:
was it necessary to remove the full url?

Reply:
Yes. Id's help the  [json-everything library](https://github.com/gregsdennis/json-everything) resolve references to local subschema files that are referenced in the [pa.yaml-schema.json](https://github.com/microsoft/PowerApps-Tooling/blob/f662d459b9d542eb26be6bd4043da42ea02430c8/docs/pa.yaml-schema.json#L26) file

Reply:
Alternatively, if a user wants to reference source schema in the repo, we could add a [json-schema comment](https://json-schema.org/understanding-json-schema/reference/comments)